core/scripts/export_submissions.sql

```diff
@@ -0,0 +1,8 @@
+-- Exports submissions and primary category from classic db.
+-- For use with process_submissions.py
+-- 
+-- mysql -u root -B arXiv < export_submissions.sql > submissions.tsv
+SELECT sub.*, cat.category
+FROM arXiv.arXiv_submissions sub, arXiv.arXiv_submission_category cat 
+WHERE sub.submission_id = cat.submission_id AND cat.is_primary = 1
```

Comment:
If the idea is to capture a wide range of submissions for testing, note that this won't capture any of the submissions that are on hold without a primary category.